lib/ecto/migration.ex

```diff
@@ -834,6 +834,7 @@ defmodule Ecto.Migration do
   end
 
   def references(table, opts) when is_binary(table) and is_list(opts) do
+    opts = opts ++ Runner.repo_config(:migration_primary_key, [])
```

Comment:
This will silently set the name of all references without name to the name of the primary key. We probably want to do this:

```elixir
repo_opts = Keyword.take(Runner.repo_config(:migration_primary_key, []), [:type])
opts = Keyword.merge(repo_opts, opts)
```

Reply:
Done.